.github/actions/backend-integration-patch-coverage/evaluate-thresholds.py

```
@@ -0,0 +1,248 @@
+
+    failed = bool(failures)
+    if failed:
+        lines.append(f"**Failed** — {len(failures)} package(s) below threshold.")
+    elif rows:
+        lines.append(f"**Passed** — {len(rows)} package(s) met their thresholds.")
+    else:
+        lines.append("**Passed** — no changed package had enough coverable lines to measure.")
+    lines.append("")
+
+    if labels:
+        lines.append(f"- Labels considered: {', '.join(f'`{l}`' for l in labels)}")
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)